### DISCUSSION TITLE
Understanding last_sensor_start_time Behavior: Seeking Clarity on Timestamp Assignment and its Impact on Job Activation

### DISCUSSION THREAD

**Discussion by laisdeghaide** [dagster-io/dagster#19004] [Community]
Hello!
Regarding the new releases, specifically `the is_first_tick_since_sensor_start` and `last_sensor_start_time` properties, I'm seeking clarification on whether `last_sensor_start_time` is assigned a timestamp only when the sensor undergoes a restart.

I have a sensor for which I intend to activate my job solely upon restart, preventing multiple evaluations on previous ticks once the sensor is powered on again. However, I've observed that in the initial tick following a restart, `last_sensor_start_time` is populated with a timestamp, but in subsequent ticks, it appears to be set to None. 

Can someone help me?

**Comment by smackesey** [comment]
cc @prha IIRC I think you added these properties?

**Comment by prha** [comment]
Hi @laisdeghaide thanks for reporting and apologies for the long delay.  I just put up [a PR](https://github.com/dagster-io/dagster/pull/19517) to patch this and will have this go out in the next release.
